From e13d946a8adb0a7f2966fc40706882f6360bedeb Mon Sep 17 00:00:00 2001
From: Andrew Purtell <apurtell@apache.org>
Date: Tue, 14 Mar 2017 13:23:11 -0700
Subject: [PATCH 329/410] HBASE-17780 BoundedByteBufferPool "At capacity"
 messages are not actionable

Ref: CDH-53383
Reviewed By: Stack

(cherry picked from commit d7666b60662cfc8492e396565b29494407c7843c)

Change-Id: Ibf181f4e2693904d4abce30e86ca87a3188614a3
---
 .../hadoop/hbase/io/BoundedByteBufferPool.java     |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/hbase-common/src/main/java/org/apache/hadoop/hbase/io/BoundedByteBufferPool.java b/hbase-common/src/main/java/org/apache/hadoop/hbase/io/BoundedByteBufferPool.java
index c685a92..07456d1 100644
--- a/hbase-common/src/main/java/org/apache/hadoop/hbase/io/BoundedByteBufferPool.java
+++ b/hbase-common/src/main/java/org/apache/hadoop/hbase/io/BoundedByteBufferPool.java
@@ -121,7 +121,9 @@ public class BoundedByteBufferPool {
       lock.unlock();
     }
     if (!success) {
-      LOG.warn("At capacity: " + this.buffers.size());
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("At capacity: " + this.buffers.size());
+      }
     } else {
       if (average > this.runningAverage && average < this.maxByteBufferSizeToCache) {
         this.runningAverage = average;
-- 
1.7.9.5

